docs/api/create_docker_container.md

```diff
@@ -0,0 +1,78 @@
+| `WithDockerEndpoint`                    | Sets the Docker daemon socket to connect to.                                                                    |
+| `WithAutoRemove`                        | Will remove the stopped container automatically like `--rm`.                                                    |
+| `WithCleanUp`                           | Will remove the container automatically after all tests have been run.                                          |
+| `WithLabel`                             | Applies metadata to the container e.g. `-l, --label testcontainers=awesome`.                                    |
+| `WithResourceReaperSessionId`           | Assigns a Resource Reaper session id to the container.                                                          |
+| `WithRegistryAuthentication`            | Sets the basic authentication against a private Docker registry.                                                |
+| `WithImage`                             | Specifies an image to derive the container from.                                                                |
+| `WithImagePullPolicy`                   | Specifies an image pull policy to determine when an image is pulled e.g. `--pull "always"\|"missing"\|"never"`. |
+| `WithName`                              | Sets the container name e.g. `--name testcontainers`.                                                           |
+| `WithHostname`                          | Sets the container hostname e.g. `--hostname testcontainers`.                                                   |
+| `WithMacAddress`                        | Sets the container MAC address e.g. `--mac-address "00:80:41:ae:fd:7e"`.                                        |
+| `WithWorkingDirectory`                  | Specifies or overrides the `WORKDIR` for the instruction sets.                                                  |
+| `WithEntrypoint`                        | Specifies or overrides the `ENTRYPOINT` that runs the executable.                                               |
+| `WithCommand`                           | Specifies or overrides the `COMMAND` instruction provided in the Dockerfile.                                    |
+| `WithEnvironment`                       | Sets an environment variable in the container e.g. `-e, --env "MAGIC_NUMBER=42"`.                               |
+| `WithExposedPort`                       | Exposes a port inside the container e.g. `--expose=80`.                                                         |
+| `WithPortBinding`                       | Publishes a container port to the host e.g. `-p, --publish 80:80`.                                              |
+| `WithBindMount`                         | Binds a path of a file or directory into the container e.g. `-v, --volume .:/tmp`.                              |
```

Comment:
```suggestion
| `WithBindMount`                         | Binds a path of a file or directory into the container e.g. `--volume .:/tmp`.                              |
```

I always removed the shorthand flag from the examples, to have the correct flag usage displayed.

Reply:
Then I would go with `-v`, `--volume`. It might sound hilarious, but not all devs know the difference. Or which shorthand flag belongs to the long name.